### DISCUSSION TITLE
weewx-GTS

### DISCUSSION THREAD

**Discussion by OE3MNS** [Daveiano/weewx-wdc#208] [Q&A]
Hallo, im Skin werden die Daten vom GTS Plugin nicht angezeigt. Muss man im Skin noch etwas ändern? Vielen Dank für eure mithilfe.
Schöne Grüße aus Tulln.

**Comment by Daveiano** [comment]
Hallo zurück! Ich selbst habe die Erweiterung nicht im Einsatz, aber das hier gefuden: https://github.com/roe-dl/weewx-GTS#including-in-skins

Die Erweiterung stellt die `GTS`-observation zur Verfügung, sollte also mit dieser Anleitung hier https://github.com/Daveiano/weewx-wdc/wiki/Adding-new-observations einzubinden sein.

Ich arbeite bald an der nächsten Version des Skins, ich werde mir dabei weewx-GTS mit ansehen, wahrscheinlich lohnt es sich ein paar Konfig-Beispiele mit ins Wiki aufzunehmen, bzw. auch eine "tiefere Integration" einzubauen.

**Reply by OE3MNS** [reply]
Herzlichen Dank ! 
Ich freu mich schon. 

LG Michi

**Comment by Daveiano** [comment]
Hi Michi,

es waren noch einige kleine Anpassungen nötig, damit die Daten von weewx-GTS angezeigt werden, diese sind im neuen Release enthalten: https://github.com/Daveiano/weewx-wdc/releases/tag/v3.4.0

Hier https://github.com/Daveiano/weewx-wdc/wiki/Code-Snippets#using-weewx-gts gibt es auch eine Bsp.-Konfiguration wie ich sie auf meiner Demoseite verwende: https://www.weewx-hbt.de/year-2022.html

Falls du noch Ideen/vorschläge hast wie man die Daten von weewx-GTS visualisieren könnte, lass es mich gerne wissen!
